scylla.go

```diff
@@ -290,15 +345,15 @@ func (p *scyllaConnPicker) maybeReplaceWithLessBusyConnection(c *Conn) *Conn {
 		return c
 	}
 	alternative := p.leastBusyConn()
-	if alternative == nil || alternative.AvailableStreams() * 120 > c.AvailableStreams() * 100 {
+	if alternative == nil || alternative.AvailableStreams()*120 > c.AvailableStreams()*100 {
 		return c
 	} else {
 		return alternative
 	}
 }
 
 func isHeavyLoaded(c *Conn) bool {
-    return c.streams.NumStreams / 2 > c.AvailableStreams();
+	return c.streams.NumStreams/2 > c.AvailableStreams()
```

Comment:
It looks like some code was not formatted properly before and here you gofmt-ed it. Could you move this change to a separate commit?

Reply:
If I run gofmt, this is what I get:
```
modified:   batch_test.go
modified:   cass1batch_test.go
modified:   cassandra_only_test.go
modified:   connectionpool_test.go
modified:   debug_off.go
modified:   debug_on.go
modified:   errors_test.go
modified:   events_ccm_test.go
modified:   example_lwt_batch_test.go
modified:   example_nulls_test.go
modified:   example_paging_test.go
modified:   example_set_test.go
modified:   example_udt_struct_test.go
modified:   exec_test.go
modified:   fuzz.go
modified:   host_source_gen.go
modified:   host_source_test.go
modified:   integration_cassandra_only_test.go
modified:   integration_test.go
modified:   metadata_cassandra.go
modified:   metadata_cassandra_test.go
modified:   metadata_scylla.go
modified:   metadata_scylla_test.go
modified:   recreate.go
modified:   recreate_test.go
modified:   scylla_shard_aware_port_integration_test.go
modified:   scylla_shard_aware_port_mocked_test.go
modified:   session_test.go
modified:   session_unit_test.go
modified:   stress_test.go
modified:   topology_test.go
modified:   tuple_test.go
modified:   udt_test.go
modified:   uuid_test.go
modified:   wiki_test.go
```
It seems that gofmt's usage was abandoned in this repo quite a long time ago. Upstream probably lives with it, and so I'm afraid we should, too. I'll just remove that fmt fix from this PR.

Reply:
We could run gofmt upstream and enforce it in CI. Would probably cause one-time merge conflict though.